python/variable.cpp

```diff
@@ -149,6 +149,11 @@ void bind_init_0D_list_eigen(py::class_<Variable> &c) {
 }
 
 void init_variable(py::module &m) {
+  // Needed to let numpy arrays keep alive the scipp buffers.
+  // VariableConcept must ALWAYS be passed to Python by shared_ptr or value.
+  py::class_<VariableConcept, std::shared_ptr<VariableConcept>>
```

Comment:
There is a typedef for this, `VariableConceptHandle`, maybe use that for the bindings instead, then part of the comment is redundant?

Reply:
Should maybe also prefix the Python name with `_` to mark it as private?

Reply:
> There is a typedef for this, VariableConceptHandle, maybe use that for the bindings instead, then part of the comment is redundant?

I wanted to be explicit. I can use the alias but I would still leave the comment because returning or taking a raw pointer as argument would break the bindings silently.

> Should maybe also prefix the Python name with _ to mark it as private?

Already the case?

Reply:
I'd say use the alias since this may change, who knows.